### ISSUE TITLE
Adding Meet-Up

### ISSUE THREAD

**Issue by kbell152** [godaddy/wp-contact-widgets#33]
Is there a way to add a social media site such as meetup.com to the plugin?

**Comment by fjarrett** [comment]
@kbell152 Absolutely, that is covered in the FAQ.

https://wordpress.org/plugins/contact-widgets/faq/

**Comment by kbell152** [comment]
Hi Frankie.

Well that’s what I thought so I tried to add meetup following the FAQ to my WP Child Theme but it didn’t work.  I added:

add_filter( 'wpcw_widget_social_custom_fields', function( $fields, $instance ) {

  $fields['meetup'] = [
    'icon'      => 'meetup', //See font-awesome icon slug
    'label'     => __( 'meetup', 'seminoleaudubon.com' ),
    'default'   => 'https://www.meetup.com/SeminoleAudubon-com/',
    'select'    => 'username',
    'sanitizer' => 'esc_url_raw',
    'escaper'   => 'esc_url',
    'social'    => true,
    'target'    => '_blank',
  ];

  return $fields;

}, 10, 2 ); 

to my child functions.php with no luck.

I can change fields for existing sites but I can’t seem to add meetup.  What am I missing?

Thanks for the quick response!

Ken

**Comment by EvanHerman** [comment]
Hi @kbell152,

To follow up here, it looks like the icon that you are trying to add was assigned in Font Awesome 4.7.
http://fontawesome.io/whats-new/

Our plugin loads Font Awesome version 4.5, which doesn't appear to have the meetup icon.

If you try switching the icon to something that is included in Font Awesome 4.5 or earlier, you should see the icon appear. Instead of `meetup` try the one form the FAQ - `scribd`.

We'll need to update the version of Font Awesome that is being enqueued by the plugin, and possibly introduce a filter to allow users to upgrade/downgrade the version as they would like.


Evan

**Comment by kbell152** [comment]
Actually I had noticed that if I changed to an existing icon on the plugin (i.e.facebook-official ) that it did work.

Thanks for the answer! I will wait patently.  <http://fontawesome.io/icon/facebook-official>

Ken

**Comment by jonathanbardo** [comment]
@kbell152 

While we deliberate on the upgrade path for this you can use the following code in your functions.php to add the meetup icon.

```php
function my_unique_function_name() {

	if ( wp_style_is( 'font-awesome', 'enqueued' ) ) {

		wp_deregister_style( 'font-awesome' );

		wp_enqueue_style( 'font-awesome', '//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css', [], '4.5.0' );

	}

}

add_action( 'wp_enqueue_scripts', 'my_unique_function_name', 20 );
add_action( 'admin_footer',       'my_unique_function_name', 20 );


add_filter( 'wpcw_widget_social_custom_fields', function( $fields, $instance ) {

	$fields['meetup'] = [
		'icon'      => 'meetup', //See font-awesome icon slug
		'label'     => __( 'meetup', 'seminoleaudubon.com' ),
		'default'   => 'https://www.meetup.com/SeminoleAudubon-com/',
		'select'    => 'username',
		'sanitizer' => 'esc_url_raw',
		'escaper'   => 'esc_url',
		'social'    => true,
		'target'    => '_blank',
	];

	return $fields;

}, 10, 2 );
```

That being said since 4.7.0 is backward compatible we shouldn't be too long to update.

**Comment by kbell152** [comment]
Hi Jonathan.

That work great!  I really appreciate the support.  You’ll get 5 stars from me!!

Ken

**Comment by fjarrett** [comment]
@kbell152 FYI - We just released v1.4.1 which now ships with FontAwesome 4.7 by default.
